build.gradle

```diff
@@ -11,6 +11,7 @@ repositories {
 }
 dependencies {
   compile 'redis.clients:jedis:[2.7.2,3)'
+  compile 'org.apache.commons:commons-lang3:3.0'
```

Comment:
Is there a reason you used version 3.0 instead of the most recent version? Looks like it's 3.4 right now. http://mvnrepository.com/artifact/org.apache.commons/commons-lang3